client/src/components/recentTasksController.js

```diff
@@ -0,0 +1,16 @@
+function recentTasksController($http) {
+  let self = this;
+
+  $http.get('/api/recenttasks').then(function(res) {
+    let tasks = res.data;
+    tasks = tasks.filter((item) => {
+      if (item.status == 'active') {
```

Comment:
Or you can retrieve all tasks, but filter it on tab change and do not send additional requests to server